library/input/input.css

```diff
@@ -58,16 +63,16 @@
 .input-field__icon--search {
```

Comment:
for less than 1 rem you should use px

Reply:
No, that statement is just a recommendation and is true for properties you don't want to be responsive, ex: border-radius.

Here it is being used for positioning and changes with other components as well.